examples/local-device/local-device-sc.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
-  name: local-device
+  name: openebs-device
```

Comment:
let us keep this as local-device only -- to avoid users from over-writing the default openebs-device class.

Reply:
Instead -- change the name of the storage class used in the PVC to "local-device" here: https://github.com/openebs/charts/blob/gh-pages/examples/local-device/local-device-pvc.yaml#L6

Also update the output of this example displayed in the docs here: https://github.com/openebs/openebs-docs/blob/staging/docs/uglocalpv-device.md#create-a-persistentvolumeclaim

Reply:
Done.